geopandas/tools/geocoding.py

```diff
@@ -22,7 +23,12 @@ def _get_throttle_time(provider):
         return 0
 
 
-def geocode(strings, provider=None, **kwargs):
+def geocode(
+    strings: list[str] | pd.Series,
+    provider: str | "geopy.geocoder" = None,
+    throttle_time: int | float = None,
```

Comment:
Is `throttle_time` renamed to `min_delay_seconds` (from `RateLimiter`)?

Reply:
Use the same name `geopy` is using, which is `min_delay_seconds`. The `throttle_time` was not exposed to user anyway, so it can be renamed.